project/app/api/database.py

```diff
@@ -58,135 +92,158 @@ def close(self):
         self.connection.close()
 
     def fetch_query_records(self, query: str):
-        cursor = self.connection.cursor()
-        cursor.execute(query)
-        result = cursor.fetchall()
+        """This is a custom query, that returns all the records based on customed query"""
+        conn, cursor = self.conn_curs()  # Establishes connection and cursor
+        cursor.execute(query)  # Execute Query
+        result = cursor.fetchall()  # fetches all records, we can limit # of records by put LIMIT statement in query
+        conn.close
         return result
 
```

Comment:
If this function only includes the following 3 lines, then it needs to be commented out or finished. This Function relates more to the above function that fetches all records.